### ISSUE TITLE
--help should show help, not considered to be a tag

### ISSUE THREAD

**Issue by varac** [GothenburgBitFactory/timewarrior#212]
I ran into this multiple times. Please show help when `--help` is used, and not consider it a tag.
Most cli tools respect this flag and show their help/usage.
It's especially helpful for new users.

```
$ timew track --help
Note: '"--help"' is a new tag.
Tracking "--help"
  Started 2019-02-26T08:18:35
  Current                  35
  Total               0:00:00
```

**Comment by lauft** [comment]
I agree. Timewarrior should process `--help` as expected.

**Comment by lauft** [comment]
Closed by #219
